components/Search/Hit.tsx

```diff
@@ -50,9 +51,22 @@ const TagsContainer = styled.div`
   gap: ${tokens.spacings.comfortable.small};
 `
 
+const TopContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: ${tokens.spacings.comfortable.large};
+`
+
 const StyledChip = styled(Chip)`
-  background-color: rgb(255, 218, 168);
+  background-color: var(--disappointing-vanilla);
+  color: var(--charcoal);
+`
+
+const StyledChipNew = styled(Chip)`
+  background-color: var(--white);
   color: var(--charcoal);
+  text-transform: uppercase;
+  border: 2px solid var(--disappointing-vanilla);
```

Comment:
In Figma the tag has a height of 24px. Here's an example of hard coding the height to 24px (should have used rems instead) and a line height of 1 (the same as the text size since it's all uppercase). I think the height of 22px for EDS Chip is because EDS uses `content-box` and not `border-box` for box-sizing. Would usually not suggested a hard-coded height, but in this particular case it does look better 🤔
![Skjermbilde 2023-03-27 kl  13 06 14](https://user-images.githubusercontent.com/1568252/227925481-5d2b2ed1-db49-4d8c-bf9d-43307894bad4.png)